docs/docsite/rst/userguide/rbac.rst

```diff
@@ -0,0 +1,513 @@
+
+If you do not want to allow custom roles, you can change the setting ``ANSIBLE_BASE_ALLOW_CUSTOM_ROLES`` to ``False``. This is still a file-based setting for now.
+
+New “add” permissions are a major highlight of this change. You could create a custom organization role that allows users to create all (or some) types of resources, and apply it to a particular organization. So instead of allowing a user to edit all projects, they can create a new project, and after creating it, they will automatically get admin role just for the objects they created.
+
+
+Resource access for teams
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This section provides a reference for managing team roles within individual resources as shown in the new UI and the corresponding API calls.
+
+Access the resource's **Team Access** tab to manage the team roles.
+
+.. image:: ../common/images/rbac_jt_team_access.png
+
+To obtain a list of team role assignments from the API: 
+
+::
+
+   GET <awxAPI>/role_team_assignments/?object_id=<template_id>&content_type__model=jobtemplate
+
+The columns are arranged so that the team name appears in the first column. The role name is under ``summary_fields.role_definition.name``
+
+To revoke a role assignment for a team in the API: 
+
+::
+
+   DELETE <awxAPI>/role_team_assignments/<role_id_from_list_API_above>/
+
+
+Add roles
+^^^^^^^^^^
+
+Clicking the **Add roles** button from the **Team Access** tab opens the **Add roles** wizard, where you can select the teams to which you want to add roles.
+
+.. image:: ../common/images/rbac_team_access_add-roles.png
+
+To list the teams from the service endpoint:
+
+::
+
+   GET <awxAPI>/teams
+
+
+The next step of the wizard in the controller UI is to apply roles to the selected team(s).
+
+.. image:: ../common/images/rbac_team_access_apply-roles.png
+
+To list available role definitions for the selected resource type in the API, issue the following, but replace ``content_type`` below to match the resource type:
+
+::
+
+   GET <awxAPI>/role_definitions/?content_type__model=jobtemplate
+
+
+Finally, review your selections and click **Save** to save your changes.
+
+.. image:: ../common/images/rbac_team_access_add-roles-review.png
+
+To assign roles to selected teams in the API, you must assign a single role to individual teams separately by referencing the team ID and resource ID from the controller associated with the ``object_id`` (``jobtemplate.id`` in this example):
+
+::
+
+   POST <awxAPI>/role_team_assignments/
+
+
+A message displays to confirm the changes were successfully applied:
+
+.. image:: ../common/images/rbac_team_access_add-roles-success.png
+
+In the API, the following message displays to confirm the changes were successfully applied:
+
+::
+
+   {"team": "011-323-232", "role_definition": 4, "object_id": "123-432-233"}
```

Comment:
This is not a message from the API, it is an example of what we should be sending as the payload for the POST request made above.
Also, since this is AWX it's probably better to have numbers as IDs in the example:
`{"team": 25, "role_definition": 4, "object_id": "10"}`

Reply:
Ah, thanks for the clarification!